iosApp/iosApp/ContentView.swift

```diff
@@ -167,6 +167,12 @@ struct ContentView: View {
                         nearbyVM.goBack()
                     }
                 }, content: coverContents)
+                .onAppear {
+                    viewportProvider.updateCameraState(locationDataManager.currentLocation)
+                }
+                .onChange(of: locationDataManager.currentLocation) { location in
+                    viewportProvider.updateCameraState(location)
```

Comment:
potential issue: It wasn't part of the original AC, but I think it would be worth adding the Location Services button below the search bar in hidden map mode. Otherwise, a user who has hidden the map but didn't grand location permissions the first time would be stuck on Park St